distribution/ddf/pom.xml

```diff
@@ -342,6 +344,13 @@
             <type>xml</type>
         </dependency>
         <dependency>
+            <groupId>org.codice.ddf.broker</groupId>
+            <artifactId>broker-app</artifactId>
+            <version>${project.version}</version>
+            <classifier>features</classifier>
+            <type>xml</type>
+        </dependency>
+        <dependency>
             <groupId>org.codice.ddf.resourcemanagement</groupId>
             <artifactId>resourcemanagement-app</artifactId>
             <version>${project.version}</version>
```

Comment:
I think you still need to add the `broker-app` lower down in all the feature descriptors